pulser-pasqal/pulser_pasqal/pasqal_cloud.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Allows to connect to the cloud powered by Pasqal to run sequences."""
+from typing import Dict, List, Mapping, Optional, Union
+
+import sdk
+from numpy.typing import ArrayLike
+
+from pulser import Sequence
+from pulser.devices import Device
+from pulser.register import QubitId
+
+
+class PasqalCloud:
+    """Manager of the connection to the cloud powered by Pasqal.
+
+    The cloud connection enables to run sequences on simulators or on real
+    QPUs.
+
+    Args:
+        client_id: client_id of the API key you are holding for Pasqal
+            cloud.
+        client_secret: client_secret of the API key you are holding for
+            Pasqal cloud.
+        endpoints: Optionally, Pasqal cloud connection URLs.
+        webhook: Optionally, webhook.
+    """
+
+    def __init__(
+        self,
+        client_id: str,
+        client_secret: str,
+        endpoints: Optional[sdk.endpoints.Endpoints],
+        webhook: Optional[str] = None,
+    ):
+        """Initializes a connection to the cloud."""
+        self._sdk_connection = sdk.SDK(
+            client_id=client_id,
+            client_secret=client_secret,
+            endpoints=endpoints,
+            webhook=webhook,
+        )
+
+    def create_batch(
+        self,
+        seq: Sequence,
+        jobs: List[Dict[str, Union[ArrayLike, Mapping[QubitId, int]]]],
```

Comment:
It would make sense to create a `JobParameters` class to guide the user on what to give here. It would, for example, force that there is a valid number of runs given, or that the values for the variables have valid types and format. The type hint here would then be `list[JobParameters]`